src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/packagename/Example2.java

```diff
@@ -0,0 +1,22 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="PackageName">
+      <property name="format"
+        value="^[a-z]+(\.[a-z][a-z0-9]*)*$"/>
```

Comment:
Change format to do violation on `com.puppycrawl.tools.checkstyle.checks.naming.packagename`
For example make it to demand package to start from capital letters not lowercase.

Reply:
do you want change this `value="^[a-z]+(\.[a-z][a-z0-9]*)*$"/>`
 to start with  capital letters  or what

Reply:
Yes.
`^[A-Z].....`.
Just to show that user can do anything he wants

Reply:
Done , but there is error again in CI/CD , I don't know what the reason 

```java

/*xml
<module name="Checker">
  <module name="TreeWalker">
    <module name="PackageName">
      <property name="format"
        value="^[a-z]+(\.[a-z][a-z0-9]*)*$"/>
    </module>
  </module>
</module>
*/

package com.puppycrawl.tools.checkstyle.checks.naming.packagename;

// xdoc section -- start
public class Example2 {}
// xdoc section -- end
```

Reply:
I made it, but it still can't pass CI/CD.